src/services/validation.js

```diff
@@ -0,0 +1,22 @@
+import firebaseAPI from "@/services/API/checkUser.js";
+import { sendPlio } from "./sendPlio";
```

Comment:
Use absolute path using "@" here

Reply:
even if it is in the same directory?

Reply:
Yep
Just to stay consistent
In case we change the directories in the future, it'll be helpful to know every import part is relative to the root src directory